src/components/playButton/PlayButton.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+import theme from '../../theme/theme'
+
+export const ButtonWrapper = styled.div`
+  width: 160px;
+  height: 160px;
+  background-color: ${theme.colors.indigo};
```

Comment:
Can you import the `colors` object only, if you're not using any other properties of the theme. its better for tree shaking if you only import what you need